src/project-css/app/Libraries/CustomStringHelper.php

```diff
@@ -178,17 +130,7 @@ public function ssnRedact($fileContents) {
     * @return array with filenames
     */
     public function checkForFilenames($string) {
-      $fileExtensions = array(
-          'txt',
-          'doc',
-          "docx",
-          "pdf",
-          "xls",
-          "xlsx",
-          "ppt",
-          "pptx",
-          "jpg"
-      );
+      $fileExtensions = array("txt", "doc", "docx", "pdf", "xls", "xlsx", "ppt", "pptx", "jpg");
```

Comment:
create a table for allowed file extensions for this and pull those that way.

Reply:
created a issue for this will work on this in a separate branch.